### ISSUE TITLE
I can not install the tool

### ISSUE THREAD

**Issue by MaryGoAround** [Nik-Zainal-Group/signature.tools.lib#6]
Hi

I have the mentioned packages one by one but again I am getting errors like

```
Error in tabToSNVcatalogue(subs = tmpSNVtab, genome.v = "hg19") : 
  could not find function "tabToSNVcatalogue"
> devtools::test()
Error: Could not find package root, is your working directory inside a package?
> 
> library(signature.tools.lib)
Error in library(signature.tools.lib) : 
  there is no package called ‘signature.tools.lib’
> 
```

How then I install your package then?

Thanks for any help

**Comment by andreadega** [comment]
Hi there,

have you followed these steps in R?
1) change directory to the signature.tools.lib git folder (use ```setwd("path/to/folder")```)
2) type ```devtools::install()```
3) check if any error appears. Sometimes there are problems installing packages that can be solved by installing some package manually.

If the installation finished properly, you should not get any error when typing ```library(signature.tools.lib)```, and should be able to run ```devtools::test()```, which also needs to be run from the signature.tools.lib git folder.

**Comment by beginner984** [comment]
Thank you

Says that

```
> getwd()
[1] "/Users/fi1d18/Downloads/signature.tools.lib-master"
> devtools::install()
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                         
2: CRAN packages only          
3: None                        
4: Rcpp (1.0.3 -> 1.0.4) [CRAN]

Enter one or more numbers, or an empty line to skip updates:
1
BSgenome.... (NA    -> 1.4.1) [CRAN]
BSgenome.... (NA    -> 1.4.0) [CRAN]
BSgenome.... (NA    -> 1.4.0) [CRAN]
Rcpp         (1.0.3 -> 1.0.4) [CRAN]
Skipping 1 packages not available: NNLM
Installing 5 packages: BSgenome.Hsapiens.UCSC.hg38, BSgenome.Mmusculus.UCSC.mm10, BSgenome.Cfamiliaris.UCSC.canFam3, NNLM, Rcpp
Error: (converted from warning) package ‘NNLM’ is not available (for R version 3.6.3)
>
```

I tried manually for NNLM
```
clang: error: unsupported option '-fopenmp'
make: *** [RcppExports.o] Error 1
ERROR: compilation failed for package ‘NNLM’
* removing ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/NNLM’
Error: Failed to install 'NNLM' from GitHub:
  (converted from warning) installation of package ‘/var/folders/wk/qwgr9yqs6ts0czx__y_w8gch0000gq/T//RtmpYIkmAy/filea1871485e02/NNLM_0.4.4.tar.gz’ had non-zero exit status
> 
```

**Comment by andreadega** [comment]
Hi,

We also had some problems with NNLM not always being available on CRAN. The
solution is usually to manually install NNLM, there should be archived
versions online that you can download.

Sometimes, errors in manual installations can be due to incompatible
versions, either of NNLM or R.

I'm sorry I can't be more helpful, this is an R installation issue, not
really a problem with our package.
